### ISSUE TITLE
Would the MS1MV3 dataset and the sub-center arcface implementation  be available?

### ISSUE THREAD

**Issue by PigletOS** [deepinsight/insightface#1243]
@jiankangdeng  @nttstar 
I lately read your paper 'Sub-center ArcFace: Boosting Face Recognitionby Large-scale Noisy Web Faces'. I wonder  if MS1MV3 and the code will be released?

**Comment by mlourencoeb** [comment]
Code is already here, under the recognition folder.

**Comment by xiaosongshine** [comment]
MS1M-V2 means MS1M-ArcFace, MS1M-V3 means MS1M-RetinaFace

**Comment by rustequal** [comment]
Unfortunately, MS1M-RetinaFace (MS1M-V3) can only be downloaded by residents of China. In order to download a 24GB file via "BaiduNetdisk", you need a Chinese phone number to register with Baidu. I could not find another way to download that file.

**Comment by AGenchev** [comment]
@rustequal yes, but they support MS1M on the academic torrents (cropped and aligned versions, TSV format), which you probably can use to produce MS1M-RetinaFace.
